### ISSUE TITLE
elFinder version 2.x

### ISSUE THREAD

**Issue by DenisAlliswell** [zxbodya/yii2-elfinder#2]
Hey @zxbodya !

First of all, thanks for this widget!

What about updating elFinder plugin to version 2.x?

**Comment by zxbodya** [comment]
Last time, I tried to use it - it was not stable enough, to be used in production... It was a while ago, and for Yii 1.x https://bitbucket.org/z_bodya/yii-elfinder2

Not sure about, elFinder 2.x state right now.

**Comment by DenisAlliswell** [comment]
@zxbodya yes, now it's stable) https://github.com/Studio-42/elFinder#branches

**Comment by Ma3oBblu** [comment]
please can you make extension with 2.x version? just like this, with composer install and etc.?

**Comment by zxbodya** [comment]
It may require significant effort to implement this and to ensure that it works with my tinymce extension(yii2-tinymce).

I am not actively developing this extension, and do not plan to implement this in foreseeable future.
